pkg/server/server.go

```diff
@@ -96,15 +96,20 @@ func (k *KiamServer) GetPodCredentials(ctx context.Context, req *pb.GetPodCreden
 
 	if !decision.IsAllowed() {
 		logger.WithField("policy.explanation", decision.Explanation()).Errorf("pod denied by policy")
+		k.recordEvent(pod, v1.EventTypeWarning, "KiamRoleForbidden",
+			fmt.Sprintf("failed assuming role: %s", req.Role))
 		return nil, ErrPolicyForbidden
 	}
 
 	creds, err := k.credentialsProvider.CredentialsForRole(ctx, req.Role)
 	if err != nil {
 		logger.Errorf("error retrieving credentials: %s", err.Error())
+		k.recordEvent(pod, v1.EventTypeWarning, "KiamCredentialError",
+			fmt.Sprintf("failed retrieving credentials: %s", err))
 		return nil, err
 	}
 
+	k.recordEvent(pod, v1.EventTypeNormal, "KiamCredentialIssued", "issued credentials")
```

Comment:
I'd want to check our metrics but I have a feeling this could get called a _lot_. Does the Event API already guard against logging the same message over and over? Don't want to do something that DDoS the API server/etcd :)

Reply:
I suppose neither of these `Normal` events is necessary now I think about it. The fact the pod is running without issue shows things are normal.

Reply:
I guess if it's possible to _only_ have credentials, or an error, then its safe to remove.

One upside to include is that if people don't see any errors, or successful messages they'll know that the annotation was misspelt/missing etc.